RCS-CSM_DIGITAL_AUTOPILOT.s

```diff
@@ -397,7 +397,7 @@ KRESUME1	TCF	RESUME		# END PHASE 1
 #		MASK	BIT3
 #		ADS	RCSFLAGS
 #
-# THEREAFTER, THE ATTITUDE ERRORS GENERATED BY THE USER SHOULD BE TRANFERRED TO THE FOLLOWING LOCATIONS IN EBANK6:
+# THEREAFTER, THE ATTITUDE ERRORS GENERATED BY THE USER SHOULD BE TRANSFERRED TO THE FOLLOWING LOCATIONS IN EBANK6:
```

Comment:
`TRANSFERED` is used in [source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1010.jpg)